python/lsst/afw/table/source/source.cc

```diff
@@ -261,4 +245,4 @@ PYBIND11_PLUGIN(_source) {
     return mod.ptr();
 }
 
-}}}  // namespace lsst::afw::table
+}}}}  // namespace lsst::afw::table::<anonymous>
```

Comment:
Given it works either way I'd be happier always putting it in the anonymous namespace. The advantage is it all wrapper code goes in the same namespace, making it easier to read.